clientcompat/pycompat/clientcompat_pb2_twirp.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, server_address):
 
```

Comment:
for readability it could make sense to rename `server_address` to `base_url`, with an example like `http://localhost:8080/twirp`